### PR TITLE
Contact form formspree

### PR DESCRIPTION
**Pull request by Aqurds**
Contact section added with the following fields,

- Full name input
- Email input
- Message input
- Submit button

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# INVALID REVIEW REQUEST

Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a PR that has conflict with the main branch. Please, kindly fix that

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -282,10 +282,17 @@ <h1 class="title contact-title">
         </p>
         <div class="contact-form">
             <form action="https://formspree.io/f/xpzebrvw" method="POST">
+<<<<<<< HEAD
+                <input type="text" name="name" maxlength="30" placeholder="Lorem Ipsum" required>
+                <input type="email" name="email" placeholder="loremipsum@gmail.com" required>
+                <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here..." required></textarea>
+                <button class="project-link" type="submit">Get in touch</button>
+=======
```
Comment (DuaneDave) [inline index.html]: As a result of the resolved conflict, you are having a duplicate of the form inputs on the UI and hence when the forms are being submitted, an error is given back. Kindly resolve this in order to make the form to function correctly.

Example of an error

fsfs

Ensure to remove the below texts that came from the conflict:
- <<<<<<< HEAD
- =======
- '>>>>>>> main'

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Aqurds 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Excellent UI design
✔️ Validations are on point

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @Aqurds @erickma1 @Peter-Kibet  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 
Changes Implemented :heavy_check_mark:
Pair Programming :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**